
function Do () {

  try {
    // something.run()
    let a = 2 + 3
    throw "Tantrum"
  } catch (e) {
    console.log("Sorry")
  }

  
}

Do()